scripts/rename-tinyagentos.sh

```
@@ -0,0 +1,776 @@
+        # Restore files from tarball (includes original tinyagentos/ directory)
+        tar xzf "$BACKUP_DIR/backup.tar.gz" -C /
+        echo "Restored files from $BACKUP_DIR/backup.tar.gz"
+    fi
+    # Remove renamed taos/ directory — the tarball already restored tinyagentos/
+    if [[ -d "taos" ]] && [[ -d "tinyagentos" ]]; then
+        rm -rf taos
+        echo "Removed taos/ (original tinyagentos/ restored from backup)"
+    fi
+    # Clean up leftover renamed files: if both taos-X and tinyagentos-X exist,
+    # the taos-X is a rename artifact (the tarball restored the original)
+    for candidate in $(find . -maxdepth 3 -name 'taos*' -not -path '*/.git/*' 2>/dev/null); do
```

Shellcheck (0.11.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)